src/RecipesBase.jl

```diff
@@ -231,14 +234,14 @@ macro recipe(funcexpr::Expr)
     if !(funcexpr.head in (:(=), :function))
         error("Must wrap a valid function call!")
     end
-    if !(isa(func_signature, Expr) && func_signature.head == :call)
+    if !(isa(func_signature, Expr) && (func_signature.head == :call || (func_signature.head == :where && func_signature.args[1].head == :call)))
```

Comment:
Great, thanks! Why don't you make a PR on @daschw 's branch so you get credit if he uses the fix?